src/core/namespace.c

```diff
@@ -1626,6 +1627,7 @@ static int mount_image(
                                        "Failed to mount image %s, extension-release metadata does not match the lower layer's: ID=%s%s%s%s%s%s%s",
                                        mount_entry_source(m),
                                        rdata.os_release_id,
+                                       rdata.os_release_id_like ? " ID_LIKE=" : "",
```

Comment:
format string above needs two more `%s` and `strempty(rdata.os_release_id_like),` needs to be added below